protocol/bc/asset.go

```diff
@@ -10,14 +10,14 @@ import (
 
 // AssetID is the Hash256 of the issuance script for the asset and the
 // initial block of the chain where it appears.
-type AssetID [32]byte
+type AssetID Byte32
```

Comment:
Would it be easier to define this in terms of Hash somehow? Or does that have to be later.